src/App/GettingStarted/getting-started/ForDevelopers/constants.js

```diff
@@ -0,0 +1,77 @@
+export const browsers = [
+    {
+        url: "img/documentation/forDevelopers/chrome.png",
+        alt: "Image of chrome",
+        text: "Yes"
+    },
+    {
+        url: "img/documentation/forDevelopers/edge.png",
+        alt: "Image of Edge",
+        text: "Yes"
+    },
+    {
+        url: "img/documentation/forDevelopers/internet-explorer.png",
+        alt: "Image of Internet explorer",
+        text: "11"
+    },
+    {
+        url: "img/documentation/forDevelopers/firefox.png",
+        alt: "Image of Firefox",
+        text: "Yes"
+    },
+    {
+        url: "img/documentation/forDevelopers/opera.png",
+        alt: "Image of Opera",
+        text: "Yes"
+    },
+    {
+        url: "img/documentation/forDevelopers/safari.png",
+        alt: "Image of Safari",
+        text: "Yes"
+    }
+];
+
+export const gridAndBreakpoints = [
+    {
+        size: "Mobile",
+        breakpoints: "320-767 px",
+        columns: "4",
+        margins: "16 px"
+    },
+    {
+        size: "Tablet",
+        breakpoints: "767-1023 px",
+        columns: "8",
+        margins: "24 px"
+    },
+    {
+        size: "Small desktop",
+        breakpoints: "1024-1439 px",
+        columns: "12",
+        margins: "24 px"
+    },
+    {
+        size: "Large desktop",
+        breakpoints: "≥ 1440 px",
+        columns: "12",
+        margins: "24 px"
+    }
+];
+
+export const discoverMore = [
```

Comment:
Might be a good idea to find a way to get URLs from one single source, to avoid dead links in case of route changes